### PR TITLE
Install gluonts for style checks.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 holidays>=0.9
 matplotlib~=3.0
-numpy~=1.16
+numpy~=1.21
```
Comment (jaheba) [inline requirements/requirements.txt:3]: We probably want to split this change into another PR.

Comment (huibinshen) [reply requirements/requirements.txt:3]: Is this the reason why `test_empirical_distribution` fails? If yes, totally.

Comment (jaheba) [reply requirements/requirements.txt:3]: No, I don't think these are related.

**Review by huibinshen — DISMISSED** [review]
Thanks for fixing these type checks!

**Review by huibinshen — APPROVED** [review]
